banzai_floyds/dbs.py

```diff
@@ -0,0 +1,38 @@
+from banzai.dbs import Base
+from sqlalchemy import Column, Integer, String, Float
+from banzai.dbs import get_session
+from astropy.coordinates import SkyCoord
+from astropy import units
+
+
+def get_standard(ra, dec, db_address, offset_threshold=5):
```

Comment:
Will this cause any issues with reduction and someone getting their data if they coincidentally looks within 5" of a standard or purposefully try to observe a standard star for science?
I remember this being an issue with the current pipeline (but I think the threshold was like 5 degrees instead of 5"). I believe we put in a proposal check before setting it as a standard.

Reply:
I'm not super opposed to doing a proposal check except that we have passed the responsibility of scheduling standards around from snex to the standards proposal. Given the small area and the fact that it is close to a bright star I think the number of false standard ids will be zero in practice.